src/Builders/ServersBuilder.php

```diff
@@ -13,7 +15,26 @@ class ServersBuilder
     public function build(array $config): array
     {
         return collect($config)
-            ->map(static fn(array $server) => Server::create()->url($server['url']))
+            ->map(static function (array $server) {
+                $variables = collect(Arr::get($server, 'variables'))
+                    ->map(function (array $variable, string $key) {
+                        $tmp = ServerVariable::create($key)
```

Comment:
`$tmp` is not a great variable name, `$serverVariable` is probably better.

Reply:
Agree